lib/player.js

```diff
@@ -5728,8 +5729,15 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
       // Bad stream?
       return;
     }
-    const liveSyncMaxLatency = this.config_.streaming.liveSyncMaxLatency;
-    const liveSyncPlaybackRate = this.config_.streaming.liveSyncPlaybackRate;
+
+    // Should use serviceDescription if they exist.
+    const liveSyncMaxLatency = this.manifest_.serviceDescription.latency ?
```

Comment:
This method is used for src= also, so this.manifest can be null. Please add more checks